apps/remote_control/lib/lexical/remote_control/build/file.ex

```diff
@@ -0,0 +1,137 @@
+defmodule Lexical.RemoteControl.Build.File do
```

Comment:
wasn't this supposed to be `RemoteControl.Build.Compile.File`?
Where's `RemoteControl.Build.Compile.Project` ?

Reply:
> Where's RemoteControl.Build.Compile.Project ?

This PR's change is mainly related to *File** compilation, **Project** compilation not changed too much, so if we want, we can extract the project compilation logic in another PR.

> RemoteControl.Build.Compile.File?

I tried this, but don't you think `Build.Compile.File.compile(document)` is slightly strange? two `compile` here.

Reply:
yeah, but it's about having some consistent type of interface between file and project operations.

If the naming isn't great, then maybe we have `Build.File.compile` and `Build.Project.compile` ?

Reply:
`Build.File.compile` yeah, we already did this.

Reply:
well, `Build.Project.compile` isn't there though.